mscore/startupWizard.cpp

```diff
@@ -0,0 +1,81 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//  Copyright (C) 2017 Werner Schweer and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+
+#include "startupWizard.h"
+#include "musescore.h"
+#include "libmscore/mscore.h"
+#include "shortcut.h"
+
+namespace Ms {
+QString StartupWizard::layoutInput()
+      {
+      QStringList layoutList;
+      QMap<QString, QString> layoutToShortcut = StartupWizard::getShortcutMap();
+      QList<QString> languages = layoutToShortcut.keys();
+      QString lang;
+      foreach(lang, languages) {
```

Comment:
Also the variable naming seems odd, it is not about languages but about keyboard layouts, isn't it?